clients/python-wrapper/lakefs/__init__.py

```diff
@@ -2,10 +2,22 @@
 Allow importing of models from package root
 """
 
+from lakefs.client import Client, DEFAULT_CLIENT
 from lakefs.repository import Repository
 from lakefs.reference import Reference
 from lakefs.models import Commit, Change, RepositoryProperties
 from lakefs.tag import Tag
 from lakefs.branch import Branch
 from lakefs.object import StoredObject, WriteableObject, ObjectReader
 from lakefs.object_manager import ObjectManager
+
+
+def repository(repository_id: str) -> Repository:
```

Comment:
Creating these  is probably out of scope for this pr. But doing so manually is a recipe for partial implementation, we're bound to miss some top level class.

Reply:
I'm not sure I understand the comment. 
We are doing this only for the Repository class, and just to align the syntax

Reply:
The comment is twofold:
1. This change does not belong on this PR: it could be split off entirely from it.  If you did that you would speed up acceptance of at least one of the two smaller PRs.
1. This is a _partial_ change that is even more confusing than providing lowercase versions of _all_ constructors.  After all, you don't do the same for any of the other imported constructors.  If you're going to do it, do it automatically and for all connstructors.